src/Form/Input.js

```diff
@@ -65,6 +85,29 @@ const AutogrowShadow = createComponent({
   }),
 });
 
+const LeftIcon = createComponent({
+  name: 'LeftIcon',
+  as: Icon,
+  style: css`
+    position: absolute;
+    left: 8px;
+    top: 50%;
+    transform: translateY(-50%);
+  `,
+});
+
+const RightIcon = createComponent({
+  name: 'LeftIcon',
+  as: Icon,
+  style: css`
+    position: absolute;
+    right: 8px;
+    top: 50%;
+    transform: translateY(-50%);
+    cursor: pointer;
```

Comment:
Observation - `RightIcon` has a `cursor:pointer` but not `LeftIcon`. Is this intended?

Reply:
Well, same thing goes here as for the search input default size. We could leave as default no cursor changes, and allow that to be determined in app.

Reply:
Going to leave the cursor style up to user.